### PR TITLE
add unpinning buttons and clear button to pin list

### LINKED ISSUE #438: the pinned items should have a button next to them for easy unpinning
It would be nice if the pinned items had a button next to them for easy unpinning, and an "unpin all" feature would be even nicer.

_Originally posted by @michaelficarra in https://github.com/tc39/ecmarkup/issues/399#issuecomment-996357637_

### PR DESCRIPTION
**Pull request by michaelficarra**
Fixes #438.

image

### CODE REVIEW

File: css/elements.css [hunk 1]
```diff
@@ -1285,12 +1326,16 @@ li.menu-search-result-term:before {
...
+  cursor: row-resize;
+}
+
 #references-pane-container.active {
   display: flex;
 }
 
 #references-pane-close:after {
-  content: '✖';
+  content: '\2716';
```
Comment (michaelficarra) [inline css/elements.css:1338]: This is a drive-by fix for an encoding error I got when doing local testing from a `file://` URL.

Comment (bakkot) [reply css/elements.css:1338]: ... is that seriously how one spells hex escapes in CSS? Gross.

Comment (michaelficarra) [reply css/elements.css:1338]: yep https://drafts.csswg.org/css-syntax/#escaping

**Review by bakkot — APPROVED** [review]
Very nice.
